### ISSUE TITLE
Feature request - increase/decrease brightness with mouse wheel

### ISSUE THREAD

**Issue by maros-ross99** [davidhi7/ddcci-plasmoid#26]
First of all, you did a great job with your widget, it should definitely be included in plasma by default! I was looking for a widget like this for so long, tried many, but none of them worked like yours. Very simple and easy but does its job perfectly!

I have a feature request that's common for plasma widgets. It's the ability to increase/decrease the value by pointing the mouse to the tray icon and scrolling with the mouse wheel. We see this feature in the audio devices widget or the battery and brightness widget so it would be really beneficial to have it standardized in your widget also. It saves from unnecessary clicking :-)

**Comment by davidhi7** [comment]
This is something I plan on implementing at some point, however it is not as trivial as it might seem as soon as you're dealing  with multiple monitors with different brightness levels.

**Comment by maros-ross99** [comment]
Definitely, it's not trivial to implement it universally. Maybe one way could be to control the brightness of the monitor where the widget is, assuming the user has multiple widgets on each monitor. Another way is to allow the wheel when the user has only one monitor, I guess this will be the most common use case.

**Comment by davidhi7** [comment]
I am not really satisfied with either solution. Similar tools for other platforms like Lunar for macOS handle this in a more intelligent way (see [this video](https://files.lunar.fyi/sync-all-displays-h264.mp4)), I think we should look into how this feature works. That being said, implementing this has a pretty low priority for me right now, so I won't be able to work on this for at least the next one or two months.

**Comment by BushwaTheGoat** [comment]
I'd really like this feature, even without the synchronisation options.

There is a [related discussion](https://github.com/rockowitz/ddcutil/issues/336) on ddcutil about synchronizing monitor brightness, Maybe synchronisation will be implemented there one day?

Thoughts about some other implementations:

[Twinkle Tray](https://github.com/xanderfrangos/twinkle-tray) supports controlling synchronized brightness for multiple displays by limiting the min/max brightness of individual screens. For me, it's not an option to reduce the min/max, so I just use the scrolling wheel without calibration, which is kinda ok and i can still adjust each screen if needed.
A similar configuration would be great, but with the ability to increase the brightness further the min/max of each screen.
![grafik](https://github.com/davidhi7/ddcci-plasmoid/assets/145575998/368e111b-1bb6-4ba5-b5f1-85a723efa397)

[MonitorControl](https://github.com/J-u-n-o/MonitorControl) allows to calibrate the screens in detail, but I think this is overkill.
![setup](https://github.com/davidhi7/ddcci-plasmoid/assets/145575998/52d24235-3557-4e25-b9b9-8fc7341b59c6)

**Comment by calvertdw** [comment]
For me, I have slightly different models of Dells in a multiple monitor setup. To get a consistent brightness across them, it requires one being set at a relative higher brightness than the others. It would be epic it the scroll wheel would adjust the values relatively, such that with a single scroll we could adjust the relative brightness of all monitors.

Thanks for the awesome plugin! It's already a huge benefit.
